test/public/envs/overview.test.js

```diff
@@ -159,7 +161,7 @@ module.exports = () => {
             el.value = '1111';
             el.dispatchEvent(new Event('input'));
         });
-        await waitForTimeout(100);
+        await page.waitForSelector(`${amountSelectorId} input:invalid`);
         expect(Boolean(await page.$(`${amountSelectorId} input:invalid`))).to.be.true;
```

Comment:
waitForSelector implies that the boolean will be true, no need for the second check

Reply:
Done